packages/checkout-storefront/src/sections/Summary/useSummaryHeightCalc.ts

```diff
@@ -35,9 +39,7 @@ export const useSummaryHeightCalc = ({ linesCount, onBreakpointChange }: UseSumm
     };
 
     window.addEventListener("resize", handleWindowResize, { passive: true });
-    handleWindowResize();
-
-    return () => window.removeEventListener("resize", handleWindowResize);
```

Comment:
This needs to stay – otherwise, we're adding a new event listener every time `[linesCount, onBreakpointChange]` changes but we never remove them.

Reply:
Removed by mistake actually, changing